src/helpers/helpers.js

```diff
@@ -0,0 +1,19 @@
+import commentsFromServer from '../api/comments.json';
+import usersFromServer from '../api/users.json';
+import postsFromServer from '../api/posts.json';
+
+function getUser(userId) {
+  return usersFromServer.find(user => userId === user.id)
+      || null;
+}
+
+function getComments(id) {
+  return commentsFromServer.filter(coment => coment.postId === id)
+      || null;
```

Comment:
```suggestion
  return commentsFromServer.filter(coment => coment.postId === id);
```
filter always returns the array